.github/workflows/code_analysis.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
 
                     -
                         name: Check YAML configs files
-                        run: bin/console lint:yaml config --ansi
+                        run: bin/console lint:yaml config --ansi --parse-tags
```

Comment:
Is this part of this PR?

Reply:
yes, see https://github.com/symfony/symfony/issues/28465